search/subrecord_discoverable.py

```diff
@@ -284,8 +284,22 @@ def get_model_fields(self):
             "which fields do you want to get off the model"
         )
 
+    def sort_fields(self, fields):
+        """
+        If the fields attribute is declared, use the order specified.
```

Comment:
This method doesn't do what it's docstring does. Instead, we make sorting decisions in get fields.